doc/api/url.md

```diff
@@ -467,6 +467,7 @@ According to the WHATWG URL Standard, special protocol schemes are `ftp`,
 #### `url.search`
 
 * {string}
+* {null}
```

Comment:
```suggestion
* {string|null}
```

Reply:
That is true. Actually, it seems there may already be a mention of this in the history for the url.parse method. I will close the PR because this has been acknowledged in the docs.